src/linker.py

```
@@ -0,0 +1,118 @@
+
+def clean_label(label):
+    label = re.sub('<[^<]+?>', '', label)
+    label = re.sub(r'\s*\(.*?\)', '', label)
+    return label.strip()
+
+def expand_alias(term):
+    """Data-driven alias expansion using Wikipedia Search."""
+    term = term.strip()
+    if len(term) > 25: return term
+    try:
+        r = requests.get(WIKIPEDIA_API, params={
```

Ruff (0.15.0)

[error] Probable use of `requests` call without timeout

(S113)